torque-core/src/test/kotlin/com/workday/torque/ModuleTestParserSpec.kt

```diff
@@ -13,241 +14,259 @@ import org.jetbrains.spek.api.dsl.given
-
-            assertThat(ModuleTestParser(args).parseTestsFromModuleApks()).isEqualTo(expectedTestModules)
-        }
-    }
-
-    context("parse tests with different test annotations") {
-        val noAnnotationsTest = TestMethod("com.company.mymodule.test#testNoAnnotations",
-                                           listOf("org.junit.Test", "kotlin.Metadata"))
-        val ignoredTest = TestMethod("com.company.mymodule.test#testIgnored",
-                                     listOf("org.junit.Ignore", "org.junit.Test", "kotlin.Metadata"))
-        val mediumTest = TestMethod("com.company.mymodule.test#testMediumTest",
-                                    listOf("android.support.test.filters.MediumTest", "org.junit.Test",
-                                           "kotlin.Metadata"))
-        val flakyMediumTest = TestMethod("com.company.mymodule.test#testMultiAnnotations",
-                                              listOf("android.support.test.filters.MediumTest",
-                                                     "android.support.test.filters.FlakyTest", "org.junit.Test",
-                                                     "kotlin.Metadata"))
-        val ignoredFlakyMediumTest = TestMethod("com.company.mymodule.test#testPositiveNegativeAnnotations",
-                                                         listOf("org.junit.Ignore",
-                                                            "android.support.test.filters.MediumTest",
-                                                            "android.support.test.filters.FlakyTest", "org.junit.Test",
-                                                            "kotlin.Metadata"))
-        val testMethods = listOf(
-                noAnnotationsTest,
-                ignoredTest,
-                mediumTest,
-                flakyMediumTest,
-                ignoredFlakyMediumTest)
-        val testPackage = ApkPackage.Valid("com.company.mymodule.test")
-        val targetPackage = ApkPackage.Valid("test.test.myapplication")
-        val testRunner = TestRunner.Valid("android.support.test.runner.AndroidJUnitRunner")
-        val apkTestParser = mockk<ApkTestParser> {
-            every { getValidatedTestPackage(any()) } returns testPackage
-            every { getValidatedTargetPackage(any()) } returns targetPackage
-            every { getValidatedTestRunner(any()) } returns testRunner
-            every { getTests(any()) } returns testMethods
-        }
-        val targetApkPath = "some_app_path"
-        val testApkPath = "some_path"
-        val testModuleInfo = TestModuleInfo(ModuleInfo(testPackage, testApkPath), testRunner, ModuleInfo(targetPackage, targetApkPath))
-
-        given("no Annotations and with default Ignore NotAnnotations") {
-            val args = Args().apply {
-                testApkPaths = listOf(testApkPath)
-                appApkPath = targetApkPath
+        {
```

Comment:
might have avoided the huge diff if this indention is moved back
I know, Spek is dumb here lol

Reply:
My bad, I'll reformat this